src/main/java/lotto/domain/Winning.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import lotto.dto.WinningInfoDTO;
+
+import java.util.Arrays;
+
+public enum Winning {
+    FIRST(6, 0, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 0, 2_000_000_000),
```
요렇게 표가할수도 있더라구요 😄 
읽기가 조금은 편해지는 장점이 있어요 ㅎ